packages/dashboard/components/company-logo.tsx

```diff
@@ -7,10 +7,12 @@ import { useRouter } from 'next/router';
 export const CompanyLogo = ({ sx }: any) => {
```

Comment:
this is where the prop should be passed in.

Reply:
This is how to get the config into the page as a page prop... https://github.com/transmute-industries/api.did.actor/blob/main/pages/%5Bdid%5D/index.tsx#L9